src/plugins/intel_gpu/tests/functional/single_layer_tests/reduce.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2024 Intel Corporation
```

Comment:
does this test fails without your change? I just checked and it succeeded :(

Reply:
The graph for func test is changed. Now, it will be failed without my change.